api/models/ivr_prompt.py

```diff
@@ -5,11 +5,12 @@
 
 
 class IvrPromptQuery(BaseQuery):
-    def get_by_name(self, name):
+    def get_by_name_and_response(self, name, response):
```

Comment:
I think we can create a new function for this and use that where we need it.

Reply:
Updated.